packages/unmock-node/package.json

```diff
@@ -16,6 +16,7 @@
     "follow-redirects": "^1.7.0",
     "ini": "^1.3.5",
     "js-yaml": "^3.13.1",
+    "mitm": "^1.7.0",
```

Comment:
Woohoo!
Notice that this version does not match the one in the main `package.json` [here](https://github.com/unmock/unmock-js/blob/0afba86bc25abeb5c8cc5caafb862e70a0baaf0e/package.json)

Reply:
That version is for `@types`.  Also, I just noticed that `expect` is in the wrong `package.json` - it should be in one of the sub-projects. Will fix.